jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java

```diff
@@ -0,0 +1,82 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2019 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.start;
+
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collection;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Unit tests for class {@link Utils}.
+ *
+ * @see Utils
+ *
+ */
+public class UtilsTest{
+
+    @Test
+    public void testIsNotBlankReturningTrue() {
+        assertTrue(Utils.isNotBlank(" @t;3/|O)t"));
+    }
+
+    @Test
+    public void testIsNotBlankReturningFalse() {
+        assertFalse(Utils.isNotBlank(null));
+    }
+
+    @Test
+    public void testIsBlank() {
+        assertFalse(Utils.isBlank(" i3o0e!#4u%QW"));
+    }
+
+    @Test
+    public void testIsBlankUsingNull() {
+        assertTrue(Utils.isBlank(null));
+    }
+
+    @Test
+    public void testJoin() {
+        assertEquals("", Utils.join((Collection<?>) null, "V9ewe2K"));
```

Comment:
This is a curious interaction as well.
I wonder if we should just use `java.util.StringJoiner` instead and deprecate this `Utils.join` method entirely?